### PR TITLE
Armanda's first blog - on context

### PR DESCRIPTION
**Pull request by MissArmieJ**
Could you all please review my blog? Your comments will be much appreciated :-)

### CODE REVIEW

File: _i18n/en.yml [hunk 1]
```diff
@@ -129,7 +129,7 @@ company-contact:
    title: Contact us
 company-contact-form:
    title: Get In Touch
-   subtitle: Call us on <a href="tel:+442074902967">+44 207 4902967</a>, email us at <a href="mailto:hello@codurance.com">hello@codurance.com</a>, or use the form below to send us a message.
+   subtitle: Call us on +44 207 4902967, email us at <a href="mailto:hello@codurance.com">hello@codurance.com</a>, or use the form below to send us a message.
```
Comment (srodrigo) [inline _i18n/en.yml]: This change, and similar ones below, look like something unrelated to your blogpost. Is there a reason for having both things in the same PR?

File: docker/Dockerfile-local [hunk 2]
```diff
@@ -1,15 +1,19 @@
 FROM ruby:2.3.1
 
+RUN apt-get update && \
```
Comment (srodrigo) [inline docker/Dockerfile-local]: I haven't run the website locally in a while. Are these changes, involving Docker, related to the blog post you are creating, or could they be part of a different PR?

**Review by srodrigo — COMMENTED** [review]
Hi Armanda,
I had a brief look at your PR, and looks to me like there are 3 different things:
- your blog post - this is the main reason of the PR, as far as I understand
- a few changes to the website (telephone numbers)
- a few changes around Docker and the way we run the website locally

Maybe, for next time, it could be split in different PRs.

File: _i18n/en.yml (hunk 1 above)
Comment (MissArmieJ) [reply _i18n/en.yml]: I may have merged in changes from a different branch.. I'll look into this - thanks

File: docker/Dockerfile-local (hunk 2 above)
Comment (MissArmieJ) [reply docker/Dockerfile-local]: There is a different PR for this called "docker-changes" - think it may still be under review. Cheers
